.github/quarkusbuilditemdoc.java

```diff
@@ -89,12 +91,16 @@ private String getJavaDoc(JavaDocCapable<?> source) {
         MutableMultimap<String, Pair<Path, JavaClassSource>> multimap = Multimaps.mutable.sortedSet
                 .with(Comparator.comparing(o -> o.getTwo().getName()));
         String line;
+        private static final List<String> KEYWORDS = List.of("extends BuildItem", "extends SimpleBuildItem",
```

Comment:
Move this to a class attribute

Reply:
Do you mean put it after line 49?

Reply:
Now I have an error : "org.jboss.forge.roaster.ParserException:  Source does not represent a [JavaClassSource], instead was [JavaAnnotationImpl] - Cannot convert."

Reply:
seem to be from this line: 
JavaClassSource source = Roaster.parse(JavaClassSource.class, fileContent);
I'm trying to resolve it

Reply:
It still works after I added back " {" to the keywords

Reply:
Ah shoot, yeah, I forgot that not every .java file in this code is a class, my bad